items/weapons.md

```diff
@@ -0,0 +1,51 @@
+# Weapons
+Weapons are items that can be equipped
+
+## General Template
+As well as the base items [General Template](README.md#general-template), weapons define these:
+
+### Dice
+The dice that are rolled for an attack made with this. See [Attacks](../systems/rolls.md#attacks)
+
+### Damage Type
+The type of damage this weapon deals by default. See [Damage Types](../systems/combat.md#damage-types)
+
+### Traits
+A list of [weapon traits](#weapon-traits) this weapon has.
+
+### Range
+The maximum effective range on this weapon.
+
+# Melee weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Sword          |A sword                                                      |2      |25     |2d6    |Slashing     |1-handed                       |5ft    |
+|War-hammer     |A wooden shafted hammer with a spike on one side             |2      |25     |2d6    |Bludgeoning  |1-handed                       |5ft    |
+|Spear          |A long wooden shaft with a metal point on the end            |2      |25     |2d6    |Piercing     |1-handed, versatile, throwing  |5ft    |
+|War-axe        |An axe with a wide head                                      |2      |25     |2d6    |Piercing     |1-handed                       |5ft    |
+
+# Ranged weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Short bow      |A short bow                                                  |2      |25     |2d6    |Piercing     |2-handed                       |120ft  |
+|Crossbow       |A crossbow                                                   |2      |25     |2d6    |Piercing     |2-handed, loading              |120ft  |
+
+# Weapon Traits
+
+## 1-handed
```

Comment:
Something cannot be 2-handed and versatile.
Perhaps the trait should be: 1-handed OR 2-handed OR versatile?

Reply:
Maybe the handedness shouldn't be a trait, and is instead a fixed setting, seeing how all items will define it. I am not 100% sold on the versatile stuff yet either. Like do we just allow any 1-handed weapon to be wielded in 2 for extra effect?

Reply:
Opinion a)
I dont know. I think it takes away from those weapons being special?
Opinion b)
Make it a feat. If you have bludgeoning master, all 1 handed bludgeoning weapons are versatile to you.
Opinion c) Idk, ask ollie.